### PR TITLE
Remove DuoSecurity files.

### PR DESCRIPTION
**Pull request by floatingatoll**
These are now maintained in IT Puppet, leave a pointer to those and remove these.
IAM-857
Closes #47 and #62.

This is a signed commit replacing #62.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,10 +6,29 @@ It also includes other IAM related scripts, such as DuoSecurity scripts that may
...
 ### duo_assign_groups.py
 Automatically assign group to new users in Duo. This is used as a default Duo group for all users as Duo cannot do this
 automatically.
 
+### duo_client_purge.py
+Deprovision users when they leave (disappear in LDAP)
+
+These were transitioned into IT puppet, where they are now maintained.  This was done without realizing that there was a github repo that otherwise had them.
+
+The scripts have diverged greatly.  Since there is little need for this repo, and it stands on the verge of being obsoleted when #58 lands.
```
Comment (bkochendorfer) [inline README.md:20]: #58 ?

Comment (floatingatoll) [reply README.md:20]: #58 hasn't landed yet, and won't before this PR.

Comment (gcoxmoz) [reply README.md:20]: If you land #63 and #58, there's pretty much nothing left in this repo.

Comment (floatingatoll) [reply README.md:20]: Indeed.

**Review by gcoxmoz — APPROVED** [review]
I like it since it's a dupe of what I wrote, but that means I shouldn't be reviewing...  But if atoll liked it enough to copy it then I think this is more like an atoll approval than me approval.   aaaugh it's all twisty

**Comment by floatingatoll** [comment]
I approved your PR, and you approved my rewrite of your PR. I could have faked it but this way you catch anything I screw up in the rewrite, if any.

**Pull request merged**